packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md

```diff
@@ -0,0 +1,132 @@
+
+With embedded authentication, the user’s credentials are directly obtained by the client application and validated by API calls to Okta, generally through the customer-hosted Sign-in Widget. Based on the organization’s requirements, the client application then creates a session for user access. The client application may also exchange tokens with a Security Token Service (STS) to provide SSO access to other service providers (CRM, IT, HR, and so on). Using this deployment model, the client’s sign-in page can render native user experiences and use native platform APIs.
+
+This deployment model does require more effort to implement and includes increased risk. However, it provides a deeper scope of customization and keeps users in the application with reduced redirects to Okta.
+
+The detailed interactions at the client level between a client’s authorization server and resource owner are provided below:
+
+<!--
+See http://www.plantuml.com/plantuml/uml/
+
+Source code for direct-auth-seq-flow.png:
+
+@startuml
+skinparam monochrome true
+
+participant "Resource Owner (User)" as rs
+participant "IDX Application Client" as cl
+participant "Identity Server" as as
+
+rs -> cl: Resource Owner
+cl -> as: Resource Owner Password Credentials
+as -> cl: Interaction required
+rs <-> cl: User interacts
+cl <-> as: User interacts
+as -> cl: Interaction Code
+cl -> as: Interaction Code
+as -> cl: Tokens
+@enduml
+
+ -->
+![Embedded Authentication](/img/direct-auth-seq-flow.png "Embedded Authentication sequence flow diagram")
+
+## Comparison between deployment models
+
+The type of deployment model or authentication is dependent on your individual implementation needs and client application. In general, the method of delegating user interaction (Redirect authentication) is generally preferred for many reasons spanning security to user experience.
+
+Use this table and following Q&A to understand the differences between Redirect Authentication and Embedded Authentication, and what flow works best for your application implementation:
+
+| Change                                                    | Redirect Authentication | Embedded Authentication |
+| --------------------------------------------------------- | ------------------------ | --------------------- |
+| Client application requires a redirect to an Identity Provider.   |   &#9989;     | |
+| Client application tightly coupled with authentication methods and processes.  |        | &#9989; |
+| SSO is implicit (if session created, SSO implemented for other resources)   |   &#9989;     | |
```

Comment:
```suggestion
| SSO is implicit (if Okta session is created, SSO is implemented for other resources)   |   &#9989;     | |
```

Reply:
Not sure if the session here is referring to the Okta session or the application session? Please confirm.

Reply:
I believe it's an Okta session, but will need to confirm. Thanks.

Reply:
Confirmed with Jeff Taylor this is Okta session.